ports/nrf/boards/bluemicro833/board.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Scott Shawcroft for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "supervisor/board.h"
+#include "mpconfigboard.h"
+#include "py/obj.h"
+#include "peripherals/nrf/pins.h"
+#include "supervisor/shared/board.h"
+
+#include "nrf_gpio.h"
+
+void board_init(void) {
+    // Turn on power to sensors and neopixels.
+    nrf_gpio_cfg(POWER_SWITCH_PIN->number,
+        NRF_GPIO_PIN_DIR_OUTPUT,
+        NRF_GPIO_PIN_INPUT_DISCONNECT,
+        NRF_GPIO_PIN_NOPULL,
+        NRF_GPIO_PIN_S0S1,
+        NRF_GPIO_PIN_NOSENSE);
+    nrf_gpio_pin_write(POWER_SWITCH_PIN->number, true);
```

Comment:
I think you'll want to "never_reset" the pin here because CircuitPython will try to reset pins after a VM run otherwise.

Reply:
never reset?  do you mean that board_init/deinit gets called every time the board is reloaded and when the code is done?

How do I make sure that the pin is ON to enable the neopixel and leave it on until the user decides to turn it off explicitely?

Reply:
yeah, the commit history isn't clean is it... I am getting better with GIT but rebases are not a common thing I do.  I'll see if I can clean it up into a single commit.
What doesn't help is the fact that github "sync" also posts commit comments...